packages/notifications/demo/stories/GlobalAlertStory.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { GlobalAlert, IGlobalAlertProps } from '@zendeskgarden/react-notifications';
+
+interface IArgs extends IGlobalAlertProps {
+  anchor?: string;
+  isExternal?: boolean;
+  button?: string;
+  isBasic?: boolean;
+  'aria-label'?: string;
+  content?: string;
+  title?: string;
+  isRegular?: boolean;
+  hasAnchor?: boolean;
+  hasButton?: boolean;
+  hasClose?: boolean;
+  hasTitle?: boolean;
+}
+
+const globalAlertText = {
+  success: {
+    title: 'Success',
+    content: 'Your first tomato has been planted.'
+  },
+  warning: {
+    title: 'Warning',
+    content: 'The soil has too much water.'
+  },
+  error: {
+    title: 'Error',
+    content: 'There are bugs in the garden.'
+  },
+  info: {
+    title: 'Info',
+    content: 'Another day in the garden.'
+  }
+};
```

Comment:
Prefer strings coming straight from the `children` controls as seen with `Notification` and `Alert`. This Garden-centric content will be helpful in the upcoming website examples. However, Storybook is meant to be more utilitarian with respect to content.